.github/CODEOWNERS

```diff
@@ -37,3 +37,8 @@
 # Statuspage.io configuration is owned by the Core SRE team
 /internal/provider/statuspage.go @hashicorp/core-sre
 /internal/providersdkv2/statuspage.go @hashicorp/core-sre
+
+# all Vault Radar resources and clients are owned by the vault-scanning team
+*vaultradar*  @hashicorp/vault-scanning
+*vault_radar* @hashicorp/vault-scanning
+*vault-radar*  @hashicorp/vault-scanning
```

Comment:
Everyone else seems to be using this patter, but will this work for directories too?
The docs don't seem to clear on this.
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
https://git-scm.com/docs/gitignore#_pattern_format